dd-sdk-android/src/main/kotlin/com/datadog/android/log/internal/utils/LogUtils.kt

```diff
@@ -0,0 +1,20 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-Present Datadog, Inc.
+ */
+
+package com.datadog.android.log.internal.utils
+
+import java.text.SimpleDateFormat
+import java.util.Locale
+import java.util.TimeZone
+
+internal const val ISO_8601 = "yyyy-MM-dd'T'HH:mm:ss.SSS'Z'"
+
+@Suppress("UnsafeThirdPartyFunctionCall")
+internal fun buildSimpleDateFormat(): SimpleDateFormat =
```

Comment:
I think it is better to specify which format return value holds in the method name, currently it is not clear what pattern it will follow. Or give pattern as argument, maybe?